ispyb/model/integration.py

```diff
@@ -85,3 +91,93 @@ def __str__(self):
     ('detector_distance', 'refinedDetectorDistance'),
     ('timestamp', 'recordTimeStamp'),
 ))
+
+class UnitCell():
+  '''An object representing the parameters of the unit cell I.e unit cell edges and angles
+  '''
+
+  def __init__(self, a,b,c,alpha,beta,gamma):
+    '''Unit cell object
+
+       :param apiid: AutoProcIntegrationID
+       :param db_conn: ISPyB database connection object
+       :return: A unit cell object representing the database entry for
+                the specified AutoProcIntegrationID
+    '''
+    self._a = a
+    self._b = b
+    self._c = c
+    self._alpha = alpha
+    self._beta = beta
+    self._gamma = gamma
+
+   
+  @property
+  def a(self):
+    '''Returns dimension a of unit cell in Angstroms'''
+    return self._a
+
+  @a.setter
+  def a(self, value):
+    '''Sets dimension a of unit cell in Angstroms'''
+    self._a = value
```

Comment:
Since you've changed the constructor to take the 6 parameters directly it would probably make sense to get rid of all the _setter_ functions. This makes the _unitcell_ object immutable, which is reasonable as we currently don't allow writing to the database via model objects anyway.